build.subr

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#
+# Shell script for building Phoenix-RTOS ports
+#
+# Common functions
+#
+# Copyright 2024 Phoenix Systems
+# Author: Daniel Sawka
+#
+
+# b_port_download(baseurl, filename, [url_filename])
+b_port_download() {
+	local baseurl="$1"
+	local filename="$2"
+	local url_filename="${3:-$filename}"
+
+	if [ ! -f "${PREFIX_PORT}/${filename}" ]; then
+		if ! wget "${baseurl}${url_filename}" -O "${PREFIX_PORT}/${filename}.part" --no-check-certificate; then
+			wget "${PORTS_MIRROR_BASEURL}${filename}" -O "${PREFIX_PORT}/${filename}.part" --no-check-certificate
```

Comment:
This should be `${filename}` actually. `url_filename` is meant to be used when the original file is located under e.g. official.url/someverylonghash.tar.gz instead of a typical package-X.Y.Z file name. The mirror should serve "normalized" files, in package-X.Y.Z format, to make uploaders' life easier.

Reply:
I find it confusing. It seemed natural for me that `url_filename` is wget input filename while `filename` is wget output filename. I'm not sure that changing mirror package filename is good idea.

Reply:
Should we put files like `9fbf5e28ef86c69858f6d3d34eccc32e911c1a28b4120ff3e84aaa70cfbf1e30.tar.gz` onto the mirror?

Reply:
Not a problem for me. If you decide to keep this feature then I suggest adding parameters description.

Reply:
Added an exhaustive (I think) descriptions.